src/libraries/ProtocolFeeLibrary.sol

```diff
@@ -20,11 +20,17 @@ library ProtocolFeeLibrary {
         if (self != 0) {
             uint16 fee0 = getZeroForOneFee(self);
             uint16 fee1 = getOneForZeroFee(self);
-            // The fee is represented in bips so it cannot be GREATER than the MAX_PROTOCOL_FEE.
+            // The fee is represented in pips so it cannot be GREATER than the MAX_PROTOCOL_FEE.
```

Comment:
```suggestion
            // The fee is represented in pips and it cannot be greater than the MAX_PROTOCOL_FEE.
```

i think some of the punctuation/language is leftover from old protocol comments. this seems better to me bc its not due to the pips denomination, and is no longer a denominator so specifying GREATER in all caps is funny now haha